Sources/FingerprintJS.xcodeproj/project.pbxproj

```diff
@@ -410,7 +413,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export PATH=$PATH:/opt/homebrew/bin\n\nif which swift-format >/dev/null; then\n     swift-format lint \\\n        --configuration ${PROJECT_DIR}/swift-format.config.json \\\n        --recursive \\\n        ${PROJECT_DIR}\nelse\n    echo \"warning: swift-format not installed\"\nfi\n";
+			shellScript = "export PATH=$PATH:/opt/homebrew/bin\n\nif which swift-format >/dev/null; then\n     swift-format lint \\\n        --configuration ${PROJECT_DIR}/../.swift-format \\\n        --recursive \\\n        ${PROJECT_DIR}\nelse\n    echo \"warning: swift-format not installed\"\nfi\n";
```

Comment:
You reviewed the PR before it was rebased and some of the comments are not outdated, but I just removed the `Format Code` build phase entirely, since we agreed to use alternative solution that won't interfere with Xcode editor. We also agreed that this solution will be implemented as part of the separate PR.

Reply:
Commenting on the non-rebased version was my fault, sorry for that. I'm not sure we agreed on removal of the linting phase from the project. We've talked about the format phase which interfered with your workflow and that's OK to remove. 90% of the comment still stands, we have 2 scripts that differ in ~2 lines of code (`lint_code.sh` and `format_code.sh`) that could be reused. The linting phase probably shouldn't have been removed at all because it helps catching errors during development.

Reply:
As agreed, I will restore the code linting build phase in a separate PR and we will leave the Bash scripts in their current shape.